src/components/TodoItem.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useRef, useEffect } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  handleUpdate: (todo: Todo) => void;
+  wantChangeTitle: number;
+  setWantChangeTitle: (id: number) => void;
+  changeTodoTitle: string;
+  setChangeTitle: (arg: string) => void;
+  handleKeyChangeTitle: (
+    event:React.KeyboardEvent<HTMLInputElement>,
+    todo: Todo) =>
+  Promise<void>;
+  handleRemoveTodo: (id: number) => void;
+  activeTodoId: number;
+};
+
```

Comment:
```suggestion
type Props = {
  todo: Todo;
  onUpdate: (todo: Todo) => void;
  wantChangeTitle: number;
  onSetWantChangeTitle: (id: number) => void;
  changeTodoTitle: string;
  onSetChangeTitle: (arg: string) => void;
  onKeyChangeTitle: (
    event:React.KeyboardEvent<HTMLInputElement>,
    todo: Todo) =>
  Promise<void>;
  onRemoveTodo: (id: number) => void;
  activeTodoId: number;
};
Follow [these](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) naming conventions for props.
```

Reply:
I added louder on edit title and  and added a condition to remove the todo if the edited title will empty